donor-portal/src/pages/about-us/index.tsx

```diff
@@ -10,6 +10,7 @@ import wso2logo from "./images/wso2.png";
 import "./styles.css";
 
 export default function AboutUs() {
+  window.scrollTo({ top: 0, left: 0, behavior: "smooth" });
```

Comment:
This will call this function on each re-render. 

A proper fix would be something like this:
https://stackoverflow.com/questions/36904185/react-router-scroll-to-top-on-every-transition

Reply:
You can try something like this:

Add the following hook to the app.tsx file. So that it will scroll to the top on every path change. 

const { pathname } = useLocation();

  useEffect(() => {
    window.scrollTo(0, 0);
  }, [pathname]);